content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst

```diff
@@ -0,0 +1,32 @@
+============
+Cycle Counts
+============
+
+In most companies, the stock is only counted once a year. That's why by default, after making an
+inventory adjustment, the scheduled date for the next count is set on the 31th of decemeber. But
```

Comment:
- Add `in Odoo` after `adjustment`
- Replace `31th` with `31st`
- Replace `decemeber` with `December`
- Replace `But` with `However,`